system-contract-dapp-playground/src/components/contract-interaction/erc/erc-20/common/OneLineMethod.tsx

```diff
@@ -19,16 +19,26 @@
  */
 
 import { Tooltip } from '@chakra-ui/react';
+import Image from 'next/image';
 import { BsFillQuestionOctagonFill } from 'react-icons/bs';
 
 interface PageProps {
   title: string;
   outputValue: string;
   titleBoxSize: string;
   explanation: string;
+  isLoading?: boolean;
+  executeCb?: (method: any) => {};
```

Comment:
Q: what's this param for?

Reply:
The `isLoading` is mainly for UI loading effect.
The `executeCb` is the function that executes the interacting APIs

Reply:
`executeCb` isn't informative of the function it serves. Especially when you compare it to `isLoading`

Reply:
Gotcha! Will put in a more informative name for it soon